From: Chirayu Desai <chirayudesai1@gmail.com>
Date: Wed, 13 Sep 2023 04:30:47 +0530
Subject: LAST: Add build args

* Keep this patch last
* To allow building without needing anything external
* arm64, because that's what we build mainly. x64 is a one line change.

Change-Id: I7e9a271b8066f7447073d5bdcfc8e02ae1612941
---
 args.gn | 60 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 60 insertions(+)
 create mode 100644 args.gn

diff --git a/args.gn b/args.gn
new file mode 100644
--- /dev/null
+++ b/args.gn
@@ -0,0 +1,60 @@
+
+android_channel="stable"
+blink_symbol_level=1
+build_contextual_search=false
+build_with_tflite_lib=false
+chrome_pgo_phase=2
+dcheck_always_on=false
+debuggable_apks=false
+dfmify_dev_ui=false
+disable_android_lint=true
+disable_fieldtrial_testing_config=true
+enable_arcore=false
+enable_av1_decoder=true
+enable_cardboard=false
+enable_dav1d_decoder=true
+enable_hangout_services_extension=false
+enable_iterator_debugging=false
+enable_mdns=false
+enable_mse_mpeg2ts_stream_parser=true
+enable_nacl=false
+enable_openxr=false
+enable_platform_dolby_vision=true
+enable_platform_hevc=true
+enable_remoting=false
+enable_reporting=false
+enable_vr=false
+exclude_unwind_tables=false
+ffmpeg_branding="Chrome"
+icu_use_data_file=true
+is_cfi=false
+#use_relative_vtables_abi=true
+is_component_build=false
+is_debug=false
+is_official_build=true
+proprietary_codecs=true
+rtc_build_examples=false
+symbol_level=1
+#system_webview_package_name="org.bromite.webview"
+target_os="android"
+treat_warnings_as_errors=true
+use_cfi_cast=true
+use_debug_fission=true
+use_errorprone_java_compiler=false
+use_official_google_api_keys=false
+use_rtti=false
+use_stable_package_name_for_trichrome=false
+#use_sysroot=false
+
+# certs/chromium
+trichrome_certdigest = "d674c702c9415fb3b6ee9a1a80b8c45ad6ea5395379eb21f51122938c15f6be8"
+
+# Need to migrate to own namespace, but without losing user data
+chrome_public_manifest_package = "org.chromium.chrome"
+system_webview_package_name = "com.android.webview"
+trichrome_library_package = "org.chromium.trichromelibrary"
+
+android_default_version_code = "672302700"
+android_default_version_name = "130.0.6723.27"
+
+target_cpu="arm64"
-- 

